jquery.noty/index.d.ts

```diff
@@ -11,51 +11,60 @@ interface NotyOptions {
     theme?: string;
     type?: string;
     /** Text to show. Can be html or string. */
-    text?: string; 
+    text?: string;
     /** If you want to use queue feature set this true. */
-    dismissQueue?: boolean; 
+    dismissQueue?: boolean;
+    /** adds notification to the beginning of queue when set to true */
+    force?: boolean;
+    /** You can set max visible notification for dismissQueue true option */
+    maxVisible?: number;
     /** The note`s optional template like '<div class="noty_message"><span class="noty_text"></span><div class="noty_close"></div></div>' */
-    template?: string; 
-    animation?: NotyAnimationOptions;
+    template?: string;
     /** Delay for closing event. Set false for sticky notifications */
-    timeout?: any; 
-    /** Adds notification to the beginning of queue when set to true */
-    force?: boolean; 
+    timeout?: any;
+    /** displays a progress bar */
+    progressBar?: boolean;
+
+    animation?: NotyAnimationOptions;
+    /** ['click', 'button', 'hover', 'backdrop'] // backdrop click will close all notifications */
+    closeWith?: any[];
```

Comment:
Can you make this `Array<'click' | 'button' | hover' | 'backdrop'>` or `('click' | 'button' | hover' | 'backdrop')[]`